website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,767 @@
+		necessary. Please see the [release highlight](/highlights/2022-04-06-disk-buffer-v2-stable) for additional
+		updates about this migration.
+
+		In addition to the new features, enhancements, and fixes listed below, this release adds:
+
+		- [Support for iteration has landed in VRL](/highlights/2022-05-18-vrl-iteration-support). Now you can
+		  dynamically map unknown key/value pairs and fields that are arrays. This replaces some common use cases for
+		  the `lua` transform with the much more performant `remap` transform.
+		- [New native event codecs](/highlights/2022-03-31-native-event-codecs) for Vector. We are still rolling out the
+		  new codec support to all sinks, but this will allow sending events (logs, metrics, and traces) between Vector
+		  instances via transports like `kafka` rather than being limited to the gRPC `vector` source and sink.
+		- A new GCP PubSub (`gcp_pubsub`) source to consume events from GCP PubSub.
+		- A new `websocket` sink was added to send events to a remote websocket listener.
+		- [New VRL functions for encrypting and decrypting data.](/highlights/2022-05-24-vrl-encryption)
+		- All sources and transforms now emit consistent metrics in accordance with Vector's [component
+		  specification](https://github.com/vectordotdev/vector/blob/927fd2eeb4ee15b10f8c046f0f9347789ca1c356/docs/specs/component.md).
+		  Sinks will included in this in the next release.
+
+		We also made additional performance improvements this release increasing the average throughput by up to 50% for
+		common topologies (see our [soak test
+		framework](https://github.com/vectordotdev/vector/tree/master/soaks/tests)).
+
+		| experiment                            | Δ mean   |   Δ mean % | confidence   |
+		|---------------------------------------|----------|------------|--------------|
+		| splunk_transforms_splunk3             | 5.98MiB  |      58.22 | 100.00%      |
+		| datadog_agent_remap_blackhole_acks    | 20.48MiB |      44.54 | 100.00%      |
+		| datadog_agent_remap_blackhole         | 20.1MiB  |      43.44 | 100.00%      |
+		| splunk_hec_route_s3                   | 5.28MiB  |      35.34 | 100.00%      |
+		| syslog_regex_logs2metric_ddmetrics    | 1.84MiB  |      15.62 | 100.00%      |
+		| syslog_log2metric_splunk_hec_metrics  | 2.52MiB  |      15.59 | 100.00%      |
+		| datadog_agent_remap_datadog_logs_acks | 9.55MiB  |      15.42 | 100.00%      |
+		| datadog_agent_remap_datadog_logs      | 9.6MiB   |      15.02 | 100.00%      |
+		| http_to_http_json                     | 2.78MiB  |      13.19 | 100.00%      |
+		| syslog_humio_logs                     | 1.91MiB  |      12.23 | 100.00%      |
+		| syslog_splunk_hec_logs                | 1.85MiB  |      12.11 | 100.00%      |
+		| syslog_loki                           | 1.42MiB  |       9.53 | 100.00%      |
+		| socket_to_socket_blackhole            | -4.61MiB |     -15.96 | 100.00%      |
```

Comment:
Should we maybe make it a habit of limiting this to only showing the top 10 results? And maybe also make it a rule to remove `_acks` variants, as they are almost always nearly identical.